coordinator/src/node.rs

```diff
@@ -325,8 +315,9 @@ impl Node {
         let event_id = format!("{contract_symbol}{maturity_time}");
         tracing::debug!(event_id, "Proposing dlc channel");
         let contract_input = ContractInput {
-            offer_collateral: margin_coordinator,
-            accept_collateral: margin_trader,
+            offer_collateral: margin_coordinator - fee,
+            // the accepting party has do bring in additional margin for the fees
+            accept_collateral: margin_trader + fee,
```

Comment:
📣 I think this is wrong: the accepting party (the app) should see their margin _reduced_ not increased.

What we want to see is:

- Coordinator LN channel balance reduced by X.
- App LN channel balance reduced by Y.
- Coordinator DLC collateral = X + fee.
- App DLC collateral = Y - fee.

In the current version the app is earning the fee.

Reply:
🤔 But if we substract the fee from the collateral of the user doesn't it mean he simply needs to provide less liquidity while the coordinator has to put up the fee for the trader?

Reply:
In the current version if the DLC goes on-chain at the same price at which it was created (i.e. no price movement):

- The coordinator will get `margin_coordinator - fee`.
- The app will get `margin_trader + fee`.

Surely that's incorrect.

Reply:
As discussed offline, neither approach seems to be correct. I will revert that change and leave it as it has been before, as this should be properly addressed once we are working on #250.